src/xunit.netcore.extensions/Attributes/ConditionalClassAttribute.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit.Sdk;
+
+namespace Xunit
+{
+    [TraitDiscoverer("Xunit.NetCore.Extensions.ConditionalClassDiscoverer", "Xunit.NetCore.Extensions")]
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    public sealed class ConditionalClassAttribute : Attribute, ITraitAttribute
+    {
+        public Type     CalleeType { get; private set; }
```

Comment:
Spacing in between Type and CalleeType.

Reply:
I used this pattern based on [ConditionalFact](https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/Attributes/ConditionalFactAttribute.cs) and [ConditionalTheory](https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/Attributes/ConditionalTheoryAttribute.cs).

I don't have strong preference on the style, will change in next commit.